apps/mobile/src/screens/adoptions/adoptions.screen.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box } from 'native-base';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+import { Adoption, MyPetsList } from './compose/my-pets-list';
+
+import SafeArea from '@/components/safe-area';
+import Topbar from '@/components/topbar';
+
+const MyAdoptionsScreen = () => {
```

Comment:
```suggestion
const MyPetsScreen = () => {
```

everything that says "my adoptions" should be renamed to "my pets"